src/teleop/jetson/jetson_teleop.py

```diff
@@ -0,0 +1,149 @@
+
+        joint_a = JointState()
+        joint_a.velocity.append(
+            self.ra_config["joint_a"]["multiplier"]
+            * quadratic(deadzone(msg.axes[self.xbox_mappings["left_js_x"]], 0.15))
+        )
+        joint_b = JointState()
+        joint_b.velocity.append(
+            self.ra_config["joint_b"]["multiplier"]
+            * quadratic(-deadzone(msg.axes[self.xbox_mappings["left_js_y"]], 0.15))
+        )
+        joint_c = JointState()
+        joint_c.velocity.append(
+            self.ra_config["joint_c"]["multiplier"]
+            * quadratic(-deadzone(msg.axes[self.xbox_mappings["right_js_y"]], 0.15))
+        )
+        joint_d = JointState()
+        joint_d.velocity.append(
+            self.ra_config["joint_d"]["multiplier"]
+            * quadratic(deadzone(msg.axes[self.xbox_mappings["right_js_x"]], 0.15))
+        )
+        joint_e = JointState()
+        joint_e.velocity.append(
+            self.ra_config["joint_e"]["multiplier"]
+            * quadratic(
+                msg.buttons[self.xbox_mappings["right_trigger"]] - msg.buttons[self.xbox_mappings["left_trigger"]]
+            )
+        )
+        joint_f = JointState()
+        joint_f.velocity.append(
+            self.ra_config["joint_f"]["multiplier"]
+            * (msg.axes[self.xbox_mappings["right_bumper"]] - msg.axes[self.xbox_mappings["left_bumper"]])
+        )
+
+        self.joint_a_pub.publish(joint_a)
+        self.joint_b_pub.publish(joint_b)
+        self.joint_c_pub.publish(joint_c)
+        self.joint_d_pub.publish(joint_d)
+        self.joint_e_pub.publish(joint_e)
+        self.joint_f_pub.publish(joint_f)
+
+        hand_finger = JointState()
+        hand_finger.velocity.append(
+            self.ra_config["finger"]["multiplier"]
+            * (msg.buttons[self.xbox_mappings["y"]] - msg.buttons[self.xbox_mappings["a"]])
+        )
+        hand_grip = JointState()
+        hand_grip.velocity.append(
+            self.ra_config["grip"]["multiplier"]
+            * (msg.buttons[self.xbox_mappings["b"]] - msg.buttons[self.xbox_mappings["x"]])
+        )
+
+        self.finger_pub.publish(hand_finger)
+        self.grip_pub.publish(hand_grip)
+
+
+def main():
+    ros.init_node("teleop")
+    xbox = ros.get_param("/teleop/xbox_mappings")
+    joystick = ros.get_param("/teleop/xbox_mappings")
+    ra_config = ros.get_param("/teleop/ra_controls")
+    drive_config = ros.get_param("/teleop/drive_controls")
+    track_radius = ros.get_param("/teleop/constants/track_radius")
+    wheel_radius = ros.get_param("/teleop/constants/wheel_radius")
+
+    arm = ArmControl(xbox_mappings=xbox, ra_config=ra_config)
+    drive = Drive(
+        joystick_mappings=joystick, drive_config=drive_config, track_radius=track_radius, wheel_radius=wheel_radius
+    )
+
+    ros.Subscriber("/drive_cmd_twist", Twist, drive.teleop_drive_callback)
+    ros.Subscriber("/xbox/ra_control", Joy, arm.ra_control_callback)
+
+    ros.spin()
+
+
+main()
```

Comment:
Kind of a silly pythonic thing, but might be nice to say `if __name__ == '__main__': main()`